docs/operator/deploy.md

```diff
@@ -6,143 +6,114 @@ sidebarDepth: 2
 
 # Deployment guide
 
-This document explains in details the workflow to have a working group of drand
-nodes generate randomness. On a high-level, the workflow looks like this:
+This document explains the workflow to have a working group of drand nodes generate randomness. There are three main sections to this guide:
 
-- **Setup**: generation of individual long-term key pair and the group file and
-  starting the drand daemon.
-- **Distributed Key Generation**: each drand node collectively participates in
-  the DKG.
-- **Randomness Generation**: the randomness beacon automatically starts as soon
-  as the DKG protocol is finished.
+- generate the long-term key pairs and the group file.
+- start the daemons.
+- have each node collectively particiate in the distributed key generation (DKG).
+- genearte randomness.
 
 ## Setup
 
 The setup process for a drand node consists of the following steps:
 
-1. Generate the long-term key pair for each node
-2. Each node starts their daemon
-3. Leader starts the command as a coordinator & every participant connects to the
-   coordinator to setup the network
+1. Generate the long-term key pair for each node.
+1. Start the drand daemon on each node.
+1. The leader starts the command as a coordinator & every participant connects to the coordinator to setup the network.
 
-This document explains how to do the setup with the drand binary itself. drand
-also offers a docker image to run the setup. You can find the information for
-running with docker [here](https://github.com/drand/drand/blob/master/docker/README.md).
+This document explains how to do the setup with the drand binary itself. If you want to install drand using Docker, follow the [Docker instructions instead](/operate/docker).
 
 ### Long-term key
 
-To generate the long-term key pair `drand_id.{secret,public}` of the drand
-daemon, execute
+Each drand node needs a public and secret key to interact with the rest of the network. To generate these keys run `drand generate-keypair` followed by the address of your node:
 
 ```bash
-drand generate-keypair <address>
+drand generate-keypair drand.example.com
 ```
 
-where `<address>` is the address from which your drand daemon is reachable. The
-address must be reachable over a TLS connection directly or via a reverse proxy
-setup. In case you need non-secured channel, you can pass the `--tls-disable`
-flag.
+The address must be reachable over a TLS connection directly, or via a reverse proxy setup. If you need a non-secured channel, you can pass the `--tls-disable` flag, although this is not recommended. Disabling TLS should only really be done when running a [local deployment](/operate/local-deploy).
+
+The default location for your keys is `/home/<USERNAME>/.drand`. You can specify where you want the keys to be saved by using the `--folder` flag:
```

Comment:
I'm pretty sure it's the system home directory in which case it's not always `/home/<USERNAME>`:

```suggestion
The default location for your keys is `~/.drand`. You can specify where you want the keys to be saved by using the `--folder` flag:
```